packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -35,22 +40,34 @@ export function Previewer({
             {translate('Drop files here or')}
           </Text>
           <UploadButton
-            {...rest}
+            acceptedFileTypes={acceptedFileTypes}
+            multiple={multiple}
+            onFileChange={onFileChange}
             className={'amplify-fileuploader__dropzone__button'}
           />
         </UploadDropZone>
-        <Text fontWeight="bold">{files.length} files selected</Text>
+        <Text fontWeight="bold">
+          {files.length} {translate('files selected')}
+        </Text>
         {files?.map((file, index) => (
-          <Tracker key={index}>{file.name}</Tracker>
+          <Tracker
+            file={file}
+            hasImage={file?.type.startsWith('image/')}
+            url={URL.createObjectURL(file)}
+            key={index}
+            onChange={(e): void => onNameChange(e, index)}
+            onCancel={() => onFileCancel(index)}
+            name={allFileNames[index]}
+          />
         ))}
         <View className="amplify-fileuploader__footer">
           <View>
             <Button size="small" variation="primary" onClick={onClick}>
-              Upload {files.length} files
+              {translate(`Upload ${files.length} files`)}
```

Comment:
From customer's perspective, does it mean they have to supply translations for 

"Upload 1 files", "Upload 2 files", ...?

Reply:
Yes, I can change that, maybe split it for just Upload and Fies

Reply:
Hmm, let's not do that just yet, since languages can be very different with its ordering of words. (e.g. "Upload 1 file can be  "1 file upload" in another language). 

Let's leave it for now, but we can discuss later.

Reply:
That said, can we adjust this to say "Upload 1 file" without plural files if `files.length === 1`?